app/app.controller.js

```diff
@@ -57,4 +57,13 @@ exports.postCommentByArticleId = (req, res, next) => {
         res.status(201).send({comment})
     })
     .catch(next)
+}
+
+exports.updateArticle = (req, res, next) => {
+    const {article_id} = req.params
+    const {inc_votes} =  req.body
+    patchArticle(inc_votes, article_id).then((updateArticle) => {
+        res.status(200).send(updateArticle)
```

Comment:
We should send our response in an object on an appropriate key e.g. {updatedArticle : updatedArticle} which can be shortened to {updatedArticle}